lib/common/presentation/views/referencing/referencing.dart

```diff
@@ -0,0 +1,316 @@
+          crossAxisAlignment: CrossAxisAlignment.start,
+          children: <Widget>[
+            Row(
+              children: [
+                Image.asset(
+                  'assets/images/referencingimage.png',
+                  height: 100,
+                  width: 100,
+                ),
+                const SizedBox(width: 20),
+                const Text(
+                  'Referencing & Citation',
+                  style: TextStyle(fontSize: 40, fontWeight: FontWeight.bold),
+                ),
+              ],
+            ),
+            const SizedBox(height: 40),
+            const Text(
+              'When writing assignments, reports, research papers or theses, '
+              'you must acknowledge quotations, information and ideas taken '
+              'from other authors through a bibliography or list of '
+              'references.',
+              style: TextStyle(fontSize: 15),
+            ),
+            const SizedBox(height: 20),
+            Row(
+              children: [
+                const Text(
+                    'The Library has developed guides for the citation styles '
+                    'most commonly used within the University. Use of these '
+                    'guides is supported in the ',
+                    style: TextStyle(fontSize: 15)),
+                InkWell(
+                  onTap: () => launch(
+                      'https://policies.westernsydney.edu.au/download.php?associated=1&id=716&version=1'),
+                  child: Text('University Referencing Styles Guidelines.',
+                      style: TextStyle(
+                        fontSize: 15,
+                        color: Colors.red[900],
+                        decoration: TextDecoration.underline,
+                      )),
+                ),
+              ],
+            ),
+            const SizedBox(height: 20),
+            Row(
+              children: [
+                const Text('-   ', style: TextStyle(fontSize: 15)),
+                InkWell(
+                  onTap: () => launch(
+                      'https://library.westernsydney.edu.au/main/sites/default/files/pdf/cite_APA.pdf'),
+                  child: Text('APA 7',
+                      style: TextStyle(
+                        fontSize: 15,
+                        color: Colors.red[900],
+                        decoration: TextDecoration.underline,
+                      )),
+                ),
+              ],
+            ),
+            Padding(
+              padding: const EdgeInsets.fromLTRB(20, 0, 0, 0),
+              child: Column(
+                children: [
+                  Row(
+                    children: [
+                      const Text('• See how: ', style: TextStyle(fontSize: 15)),
+                      InkWell(
+                        onTap: () => launch(
+                            'https://www.youtube.com/watch?v=drVG9qb1wNo'),
+                        child: Text('Referencing a book',
```

Comment:
with paths such as urls, create a constants file for them as that path may be accessed later down the line. It's bad practice to just use a string like that directly